packages/popup-component/src/components/popup/popup.tsx

```diff
@@ -26,9 +27,19 @@ export class DcPopup {
    */
   @State() open: boolean = true;
 
+  componentWillLoad() {
+    const heroMessageCookie = Cookies.get('hero-message');
```

Comment:
@arathjz what about the case when the content changes but the title doesn't? Should we expire the cookie selection?

Reply:
@orlando I think that for this iteration we could work with the title, what do you say?

Btw, I tried to access to the content, do you know a way to access the content of a `slot` tag?